### DISCUSSION TITLE
Grid component with SignalR / SqlTableDependency

### DISCUSSION THREAD

**Discussion by heather-timmins** [vikramlearning/blazorbootstrap#658] [Q&A]
Hi all, 

Wonder if anyone has used SignalR with the Grid component? 

I have successfully populated the Grid component from my EF database call.  All working as expected. 
However, I am also using SignalR and SqlTableDependency to auto refresh my data when it changes in the database but this Grid component is not changing.  

I know the SqlTableDependency and SignalR refresh is working as I'm displaying the data under the grid in a foreach loop just for  testing purposes - This works and the data changes as expected. 

Thanks in advance :-)

**Comment by gvreddy04** [comment]
@heather-timmins Thank you for using **BlazorBootstrap**. Please try calling the `RefreshDataAsync()` method on the grid reference to refresh the data. If this is not working, please share a sample GitHub repository to reproduce the issue with minimal steps. I'll take a look.

**Reply by heather-timmins** [reply]
@gvreddy04  Hi! Thank you so much for your quick response - That worked :-)
